kitchen.yml

```diff
@@ -7,37 +7,230 @@ provisioner:
   product_name: chef
   enforce_idempotency: true
   multiple_converge: 2
-  solo_rb:
-    treat_deprecation_warnings_as_errors: true
+  deprecations_as_errors: true
+  data_bags_path: test/integration/data_bags
 
 verifier:
   name: inspec
+  inspec_tests:
+    - path: test/integration/inspec
 
 platforms:
-  - name: amazonlinux-2
-    driver_config:
-      box: ywatase/amzn2
-  - name: amazon-linux
-    driver_config:
-      box: mvbcoding/awslinux
-  - name: centos-6
   - name: centos-7
-  - name: debian-8
+  - name: centos-8
   - name: debian-9
-  - name: opensuse-leap-15
+  - name: debian-10
   - name: ubuntu-16.04
   - name: ubuntu-18.04
+  - name: ubuntu-20.04
 
 suites:
-  - name: client_5.6
+  - name: client-56
     run_list:
-      - recipe[test::client-56]
-  - name: client_5.7
+      - recipe[test::client]
+    attributes:
+      percona:
+        version: '5.6'
+    verifier:
+      controls:
+        - client
+        - toolkit
+      inputs:
+        version: '5.6'
+    excludes:
+      - centos-8
+      - debian-10
+      - ubuntu-20.04
+  - name: client-57
```

Comment:
should we have a return between the different suites for legibility purposes?

Reply:
What do you mean?